cql3/result_generator.hh

```diff
@@ -114,7 +114,7 @@ private:
             }
         }
 
-        uint32_t rows_read() const { return _total_row_count; }
+        uint64_t rows_read() const { return _total_row_count; }
```

Comment:
Nit: Just an idea. Maybe it would make sense to define the return type of this function as auto to prevent the need of changing it in the future?